### PR TITLE
[Android] Add buildFlavours for mainnet or testnet

### PR DESCRIPTION
**Pull request by jonas-lundqvist**

### CODE REVIEW

**Comment by cculianu** [comment]
Ah, thanks for this.  I'll page @mhsmith and get him to review it.  👍

File: android/app/src/main/java/org/electroncash/electroncash3/Addresses.kt [hunk 1]
```diff
@@ -123,7 +124,7 @@ class AddressDialog() : MenuDialog() {
             R.id.menuCopy -> {
                 copyToClipboard(
                     if (clsAddress["FMT_UI"] == clsAddress["FMT_LEGACY"]) address
-                    else "bitcoincash:" + address)
+                    else clsNetworks["CASHADDR_PREFIX"].toString()+ ":" + address)
```
Comment (mhsmith) [inline android/app/src/main/java/org/electroncash/electroncash3/Addresses.kt]: Missing space before "+".

File: android/app/src/main/java/org/electroncash/electroncash3/App.kt [hunk 2]
```diff
@@ -39,6 +39,10 @@ class App : Application() {
                                     NotificationManager.IMPORTANCE_DEFAULT))
         }
 
+        if(BuildConfig.testnet) {
+            libNetworks.callAttr("set_testnet")
+        }
+
```
Comment (mhsmith) [inline android/app/src/main/java/org/electroncash/electroncash3/App.kt]: Missing space before "(".

**Comment by mhsmith** [comment]
I'm impressed that you found a way to do this with such a small change. Just found a couple of code style issues as mentioned above. 

I'm not sure if this will work with the Docker build script which Jonald uses for each release. Do you have a Linux machine or VM which you can test this on? The process is described in `android/README.md` and is quite simple.

My guess is that adding the build flavors will change the name of the output directory from `release` to `releaseMainNet` or similar, in which case `android/build.sh` will need to be updated.

**Comment by cculianu** [comment]
Pinging @jonas-lundqvist  :)

**Comment by cculianu** [comment]
@mhsmith Let me know when you're happy and I'll merge!

**Comment by mhsmith** [comment]
Looks good to me.

**Pull request merged**